src/odemis/acq/move.py

```diff
@@ -34,10 +34,10 @@
 from odemis.util import executeAsyncTask
 
 MAX_SUBMOVE_DURATION = 60  # s
-LOADING, IMAGING, TILTED, COATING, IMAGING_PATH, COATING_PATH, UNKNOWN = 0, 1, 2, 3, 4, 5, 6
+LOADING, IMAGING, TILTED, COATING, LOADING_PATH, UNKNOWN = 0, 1, 2, 3, 4, 5
 ATOL_LINEAR_POS = 100e-6  # m
 ATOL_ROTATION_POS = 1e-3  # rad (~0.5°)
-RTOL_PROGRESS = 0.1
+RTOL_PROGRESS = 0.3
```

Comment:
Increased this value to reflect the results I got in the real stage, but may need more testing.